Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -336,6 +338,8 @@ public void Execute_TeamBuild_CoverageReportProcessorCalled()
         fileWrapper.Received().AppendAllText(
             Arg.Any<string>(),
             Arg.Is<string>(x => x.Contains("sonar.cs.vscoveragexml.reportsPaths") && x.Contains(PathCombineWithEscape("VS", "XML", "Coverage", "Path"))));
+        new ScannerEngineInputReader(scannerEngineInput.ToString()).AssertProperty("sonar.cs.vstest.reportsPaths", Path.Combine("VS", "Test", "Path"));
+        new ScannerEngineInputReader(scannerEngineInput.ToString()).AssertProperty("sonar.cs.vscoveragexml.reportsPaths", Path.Combine("VS", "XML", "Coverage", "Path"));
```

Comment:
The reader should not be created twice
```suggestion
        var reader = new ...
        reader.AssertProperty(...
        reader.AssertProperty(...
```